docs/mkdocs.yaml

```diff
@@ -0,0 +1,90 @@
+site_name: niri
+docs_dir: wiki
+site_url: https://chinatsu.github.io/niri/ # TODO before merging: https://yalter.github.io/niri
+repo_url: https://github.com/chinatsu/niri # TODO before merging: https://github.com/YaLTeR/niri
+edit_uri: edit/main/docs/wiki/
+use_directory_urls: false
+theme:
+  name: material
+  features:
+    - navigation.instant
+    - search.suggest
+    - content.code.copy
+    - content.action.edit
+  palette:
+    scheme: slate
+    primary: black
+markdown_extensions:
+  - github-callouts
+  - pymdownx.highlight:
+      anchor_linenums: true
+      line_spans: __span
+      pygments_lang_class: true
+  - pymdownx.inlinehilite
+  - pymdownx.snippets
+  - pymdownx.superfences
+  - admonition
+  - pymdownx.details
+  - pymdownx.keys
+  - attr_list
+plugins:
+  - search
+hooks:
+  - hooks/shortcodes.py
+  - hooks/remove-must-fail.py
```

Comment:
Do we use must-fail anywhere? If not maybe can remove

Reply:
We use must-fail once. Removing it made a rust test fail, so I decided to keep it and instead remove it with this script.